src/responsive-layout.css.ts

```diff
@@ -11,50 +11,75 @@ export const LARGE_DESKTOP_MAX_WIDTH = 1224;
 const sideMargin = createVar();
 export const vars = {sideMargin};
 
-export const container = sprinkles({width: '100%'});
+export const responsiveLayoutContainer = style([
+    sprinkles({width: '100%'}),
+    {
+        vars: {
+            [sideMargin]: '0px',
+        },
+        '@media': {
+            [mq.desktopOrBigger]: {
+                vars: {
+                    [sideMargin]: `${SMALL_DESKTOP_SIDE_MARGIN}px`,
+                },
+            },
+            [mq.tablet]: {
+                vars: {
+                    [sideMargin]: `${TABLET_SIDE_MARGIN}px`,
+                },
+            },
+            [mq.mobile]: {
+                vars: {
+                    [sideMargin]: `${MOBILE_SIDE_MARGIN}px`,
+                },
+            },
+        },
+    },
+]);
+
+export const expandedResponsiveLayoutContainer = style([
+    {
+        selectors: {
+            '& &': {
+                width: 'auto',
+                margin: `0 calc(-1 * ${sideMargin})`,
+                '@media': {
+                    [mq.desktop]: {
+                        margin: `0 calc(-1 * ${sideMargin})`,
+                    },
```

Comment:
isn't this mq redundant? it's like the default case

Reply:
removed